support-frontend/assets/pages/weekly-subscription-checkout/components/thankYou.tsx

```diff
@@ -95,6 +117,29 @@ function StartDateCopy({
 	return null;
 }
 
+function StartDateCopyThreeTier({ startDate }: { startDate: Option<string> }) {
```

Comment:
```suggestion
function StartDateCopyThreeTier({ startDate }: { startDate?: string }) {
```

Is `Option` scala?

Reply:
Not sure, suspect its JS way of doing the TS `?`